components/timezone/src/provider/names.rs

```diff
@@ -95,6 +95,8 @@ pub struct IanaToBcp47MapV3<'data> {
     // Note: this is 9739B as `ZeroVec<TimeZoneBcp47Id>` (`ZeroVec<TinyStr8>`)
     // and 9335B as `VarZeroVec<str>`
     pub bcp47_ids: ZeroVec<'data, TimeZoneBcp47Id>,
+    /// Metazones
+    pub metazones: ZeroVec<'data, u16>,
```

Comment:
Issue: Besides iterating over metazones, what do you plan to use this field for?

(We shouldn't add fields to the core TimeZoneIdMapper data struct unless they have a strong core use case)